src/ResourceManager/ApiManagement/Commands.ApiManagement/Commands.ApiManagement.csproj

```diff
@@ -22,7 +22,7 @@
     <DefineConstants>TRACE;DEBUG;CODE_ANALYSIS</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
@solankisamir same comment about why this was changed to `false`

Reply:
Marking a parameter/cmdlet obsolete was throwing warning, which were being treated as Build errors.

---
In reply to: [188767844](https://github.com/Azure/azure-powershell/pull/6189#discussion_r188767844) [](ancestors = 188767844)